src/plinder/eval/docking/utils.py

```diff
@@ -420,40 +427,49 @@ def get_average_posebusters(self) -> dict[str, list[Any]]:
                     avg_scores[f"posebusters_{key}"] = None
         return avg_scores
 
-    def summarize_scores(self) -> dict[str, Any]:
-        scores: dict[str, Any] = dict(
-            model=self.model.name,
-            reference=self.reference.name,
-            num_reference_ligands=self.reference.num_ligands,
-            num_model_ligands=self.model.num_ligands,
-            num_reference_proteins=self.reference.num_proteins,
-            num_model_proteins=self.model.num_proteins,
-            fraction_reference_ligands_mapped=self.num_mapped_reference_ligands
-            / self.reference.num_ligands,
-            fraction_model_ligands_mapped=self.num_mapped_model_ligands
-            / self.model.num_ligands,
-        )
-        score_list = ["lddt_pli", "lddt_lp", "bisy_rmsd"]
-        for score_name in score_list:
```

Comment:
@Ninjani - as a result of this change we are no longer getting:
`bisy_rmsd_wave` , `bisy_rmsd_ave`, `lddt_pli_wave`, etc. scores.

What is the desired behaviour. We should decide and then update:
1) the docs:
- here: https://github.com/plinder-org/plinder/blob/main/docs/evaluation.md, 
- and here: https://github.com/plinder-org/plinder/blob/main/docs/dataset.md#linked-systems-links

2) make_plots code (multiple functions):
https://github.com/plinder-org/plinder/blob/main/src/plinder/eval/docking/make_plots.py